config/cdc/postgres-entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+
+WAL2JSON_VERSION=0.0.1
+
+cdc_setup_hba_conf() {
+    # Ensure pg-hba is properly configured
+
+    PG_HBA="$PGDATA/pg_hba.conf"
```

Comment:
Where is PGDATA defined? What happens if it's not defined?

Should this configuration file be committed in our repo and use a Docker COPY step?

Reply:
PGDATA is a documented env variable of a standard postgres docker container 
https://hub.docker.com/_/postgres. (sorry cannot link directly to the right paragraph)

Since this runs in the postgres container it is guaranteed to be defined.
Which config are you talking about ? If it is about pg_hba.conf. I don't think so. Our goal is to add a line to that file if it is not there, not to replace it completely.